### PR TITLE
Log conversations to slack

### PR DESCRIPTION
**Pull request by sjkoelle**

### CODE REVIEW

File: openduck-py/openduck_py/logging/slack.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import os
+import requests
+from openduck_py.settings import IS_DEV
+from openduck_py.utils.s3 import s3_client
+
+LOGGING_BUCKET = os.environ.get("OUTPUT_BUCKET")
+SLACK_BOT_TOKEN = os.environ.get("SLACK_BOT_TOKEN")
+SLACK_LOGS_CHANNEL_ID = os.environ.get("SLACK_LOGS_CHANNEL")
```
Comment (zachwe) [inline openduck-py/openduck_py/logging/slack.py:8]: plz add to `.env.example`

File: openduck-py/openduck_py/routers/voice.py [hunk 2]
```diff
@@ -92,6 +94,13 @@ def close_file(self):
             self.file = None
             self.params_set = False
 
+    def log(self):
+        if LOG_TO_SLACK:
+            from openduck_py.logging.slack import log_audio_to_slack
```
Comment (zachwe) [inline openduck-py/openduck_py/routers/voice.py:99]: import to top

File: openduck-py/openduck_py/logging/slack.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+_client = s3_client()  # NOTE (Sam): should we move this to utils/s3?
+
+
+def log_audio_to_slack(audio_path):
+
+    s3_key = audio_path.replace("//", "_")
```
Comment (zachwe) [inline openduck-py/openduck_py/logging/slack.py:15]: remove
```suggestion
    s3_key = audio_path
```

**Review by zachwe — APPROVED** [review]

File: openduck-py/openduck_py/logging/slack.py (hunk 1 above)
Comment (sjkoelle) [reply openduck-py/openduck_py/logging/slack.py:8]: done

File: openduck-py/openduck_py/routers/voice.py (hunk 2 above)
Comment (sjkoelle) [reply openduck-py/openduck_py/routers/voice.py:99]: done

File: openduck-py/openduck_py/logging/slack.py (hunk 3 above)
Comment (sjkoelle) [reply openduck-py/openduck_py/logging/slack.py:15]: done

**Pull request merged**